invision-app/app/components/home/home.js

```diff
@@ -34,29 +31,46 @@ export class Home extends React.Component {
             movies.sort((movieA, movieB) => movieA.release_date > movieB.release_date ? 1 : -1);
         }
 
-        this.setState({
-            movies
-        });
+        setMovies(movies);
+    }
+    function createUpdateMovie(movie) {
```

Comment:
Function that do more than one thing is antipattern. Please, create **createMovie** and **updateMovie** as a separate functions

Reply:
Modified.